petpals/messaging/static/js/new_message.js

```diff
@@ -0,0 +1,70 @@
+var recipient = null;
```

Comment:
If I'm not mistaken it should be `let` since its a general rule to not use `var`.

Reply:
var sets a variable to be global.